### PR TITLE
Update WordPress Authenticator to remove Lottie dependency

### PR DESCRIPTION
**Pull request by mokagio**
See https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640.

**Testing**

As you can see, the UI tests are green and we do test the login flow extensively. On top of that, I tested:

- Direct login via the iOS password manager prompt.
- Login via login button and 1Password autofill.
- Google login.
- That the signup flow works, i.e. results in me receiving a signup-link mail at the given address. _I did not complete the signup as that happens outside of the app anyway._
- That the prologue carousel still works (unnecessary, given it's a [different class](https://github.com/wordpress-mobile/WordPress-iOS/blob/0247336ff65e2df7ec22c4af36e7fdc1c0c72a2f/WordPress/Classes/ViewRelated/NUX/UnifiedPrologueViewController.swift) than the library's, but felt like it wouldn't hurt 😅).

## Regression Notes
1. Potential unintended areas of impact. – Login and signup flow. 

2. What I did to test those areas of impact (or what existing automated tests I relied on) – See details above.

3. What automated tests I added (or what prevented me from doing so) – None

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

File: Podfile
Comment (mokagio) [inline Podfile]: Of course, before merging, we'll want at least want https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/640 to be merged and shipped as a beta or stable.

Comment (AliSoftware) [reply Podfile]: Added `[Status] DO NOT MERGE` to the PR to make that even more visible.

Comment (mokagio) [reply Podfile]: Nice. Better than just a comment 😅 Plus gets Peril to help us with it, too.

**Review by AliSoftware — APPROVED** [review]
:shipit:

**Pull request merged**
